src/cleanvision/imagelab.py

```diff
@@ -408,23 +406,15 @@ def report(
             issue_types_to_report, report_args["max_prevalence"]
         )
 
-        num_report = (
-            report_args["num_top_issues"] if issue_types is None else len(issue_types)
-        )
-
         issue_summary = self.issue_summary[
-            self.issue_summary["issue_type"].isin(filtered_issue_types[:num_report])
+            self.issue_summary["issue_type"].isin(filtered_issue_types)
```

Comment:
where are you filtering the issue-types to only include those which occur in > 0 images of this dataset?  It doesn't look to me like you `_filter_report()` function does this filtering?

This is why it would be helpful to at least add a 1-2 sentence docstring for every function saying in English what the function does.

Reply:
Check [line](https://github.com/cleanlab/cleanvision/pull/118/files#diff-a39bd3b22f6525a8f778371319f38508ceffc82ebec07df40a94957a82b39963L402) 

I'll also add a docstring.